File: python/1276.py
from typing import List

class Solution:
    def numOfBurgers(self, tomatoSlices: int, cheeseSlices: int) -> List[int]:
        if tomatoSlices % 2 == 1:
            return []
        if cheeseSlices * 2 > tomatoSlices or cheeseSlices * 4 < tomatoSlices:
            return []
        x = (tomatoSlices - 2 * cheeseSlices) // 2
        return [x, cheeseSlices - x]

File: go/1276.go
package main

func numOfBurgers(tomatoSlices int, cheeseSlices int) []int {
	if tomatoSlices%2 == 1 {
		return []int{}
	}
	if tomatoSlices < cheeseSlices*2 || tomatoSlices > cheeseSlices*4 {
		return []int{}
	}
	x := (tomatoSlices - 2*cheeseSlices) / 2
	return []int{x, cheeseSlices - x}
}
